docs/layouts/shortcodes/generated/optimizer_config_configuration.html

```diff
@@ -89,5 +89,17 @@
             <td>Boolean</td>
             <td>When it is true, the optimizer will collect and use the statistics from source connectors if the source extends from SupportsStatisticReport and the statistics from catalog is UNKNOWN.Default value is true.</td>
         </tr>
+        <tr>
+            <td><h5>table.optimizer.busy-join-reorder</h5><br> <span class="label label-primary">Batch</span> <span class="label label-primary">Streaming</span></td>
+            <td style="word-wrap: break-word;">false</td>
+            <td>Boolean</td>
+            <td>Enables busy join reorder in optimizer. Default is disabled.</td>
+        </tr>
+        <tr>
+            <td><h5>table.optimizer.busy-join-reorder-dp-threshold</h5><br> <span class="label label-primary">Batch</span> <span class="label label-primary">Streaming</span></td>
```

Comment:
for introducing public option, we should post a FLIP or a DISCUSS.

we can remove keyword `dp` from the option since both join reorder programs using dynamic programming algorithm.  just like: `table.optimizer.busy-join-reorder-threshold`

Reply:
Done!